yt_dlp/extractor/npo.py

```diff
@@ -245,7 +245,8 @@ def _real_extract(self, url):
                     'quality': 'npoplus',
                     'tokenId': player_token,
                     'streamType': 'broadcast',
-                })
+                },
+                data=b'')  # by posting empty data we force a POST request
```

Comment:
comment not necessary
```suggestion
                }, data=b'')
```